cmd/cli/kubectl-kyverno/testing/test.go

```diff
@@ -0,0 +1,1315 @@
+package testing
```

Comment:
I don't understand why you're including this file with contents copied and pasted from the documentation. This does not represent the new schema contained in the PR.

Reply:
not following you, this particular file carry the code for new implementation, which part of the documentation is copied ?